apisix/plugins/jwt-auth.lua

```diff
@@ -105,6 +106,12 @@ function _M.check_schema(conf, schema_type)
             conf.secret = ngx_encode_base64(resty_random.bytes(32, true))
         end
 
+        if conf.algorithm == "RS256" then
+            if not conf.public_key or not conf.private_key then
+                return false, "when using RS256 algorithm, the parameter 'public_key' and 'private_key' is both required."
```

Comment:
this line is long

Reply:
and we should use JSONSchema to check it.

Reply:
I deleted the check code. If I want to express the dependency between parameters, how to do it by JSONSchema. 
related https://github.com/api7/jsonschema/issues/2